Endret feil-styling og state-handling for skjema-komponenter

KenAJoh opened the pull request:
- Hvis en skjemagruppe har en feil, vil alle skjem-elementene også få feil-styling by default 
- Flere komponenter kan nå bruke `utenFeilPropagering`-prop, og ungår da feil-styling by default 
- Feil + disabled state for enkelt-elementer er ikke tillat lengre
- Feil + disabled state for element i skjemagruppe vil ha både disabled og feil-styling ([ ] Dette er sjekket med UU-team)
- `feil`-prop er lagt til for CheckboxPanel og RadioPanel
- Feilmeldings-eksempler er oppdatert for å bedre fortelle bruker hva som er forventet der

`-beta.1` versjoneringen blir automagiskt omgjort til `x.1.0` av lerna i CI/CD pipeline


<img width="406" alt="Screenshot 2021-03-05 at 12 55 55" src="https://user-images.githubusercontent.com/26967723/110112574-37dfda00-7db2-11eb-88c1-f4a6d7cd251d.png">
<img width="395" alt="Screenshot 2021-03-05 at 12 55 44" src="https://user-images.githubusercontent.com/26967723/110112578-39110700-7db2-11eb-9b06-806f539d1b2e.png">
<img width="649" alt="Screenshot 2021-03-05 at 12 55 38" src="https://user-images.githubusercontent.com/26967723/110112580-39110700-7db2-11eb-90cd-b591f3291f64.png">
<img width="661" alt="Screenshot 2021-03-05 at 12 55 27" src="https://user-images.githubusercontent.com/26967723/110112581-39a99d80-7db2-11eb-84bd-2ef698dd0a9f.png">

KenAJoh committed:
bedre feilhåndtering for skjemaelementer

KenAJoh committed:
oppdatert disabled + feil state

KenAJoh committed:
yarn lockfile

KenAJoh committed:
oppdatert stories

KenAJoh committed:
oppdatert examples

KenAJoh committed:
oppdatert yarn lock

KenAJoh committed:
oppdatert versions

KenAJoh committed:
oppdatert versions-2

KenAJoh committed:
Lerna versionbump [ci skip]

 - nav-frontend-skjema-style@2.1.0
 - nav-frontend-skjema@3.1.0

KenAJoh committed:
Update package.json

KenAJoh committed:
Revert "Lerna versionbump [ci skip]"

This reverts commit bf03cf5c6218db1a71d5b62416fb451fb90ae002.

KenAJoh committed:
Revert "oppdatert versions-2"

This reverts commit da3cc2dfdf6ea82a8504a81b689ba58dca6d6b71.

KenAJoh committed:
oppdatert version

KenAJoh committed:
Merge branch 'examples-forms' of https://github.com/navikt/nav-frontend-moduler into examples-forms

KenAJoh committed:
Merge branch 'master' into examples-forms

KenAJoh committed:
fikset state-styling

andnorda reviewed (dismissed):
Veldig god endring! Bra jobba. Vil du skrive en liten sak i changelog?

Det er en del stygg kode her, men sånn er det å jobbe med legacy. LGTM.

KenAJoh committed:
Lerna versionbump [ci skip]

 - @navikt/ds-css@0.1.18
 - @navikt/ds-react@0.1.2
 - nav-frontend-skjema-style@2.1.0
 - nav-frontend-skjema@3.1.0

KenAJoh committed:
Revert "Lerna versionbump [ci skip]"

This reverts commit ba6a958396b6fae612b7babed1760add0a5244e3.

KenAJoh committed:
Merge branch 'master' into examples-forms

KenAJoh committed:
yarn lock
